src/dotnet-dev-certs/Program.cs

```diff
@@ -95,7 +107,33 @@ public static int Main(string[] args)
             }
         }
 
-        private static int CheckHttpsCertificate(CommandOption check, CommandOption trust, IReporter reporter)
+        private static int CleanHttpsCertificates(IReporter reporter)
+        {
+            var manager = new CertificateManager();
+            try
+            {
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                {
+                    reporter.Output("Cleaning HTTPS development certificates from the machine. A prompt might get" +
+                        "displayed to confirm the removal of some of the certificates.");
+                }
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+                {
+                    reporter.Output("Cleaning HTTPS development certificates from the machine. This operation might" +
+                        "require elevated privileges. If that is the case, a prompt for credentials will be displayed.");
+                }
+                //manager.CleanHttpsCertificates();
```

Comment:
You don't actually clean- is the comment intentional?

Reply:
I'm waiting for the changes I did in common to go through